app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -1,14 +1,14 @@
 <div class="sul-embed-container media-component" id='sul-embed-object' style='display:none;' data-controller="media media-tag">
   <header>
     <nav aria-label="Window navigation">
-      <button data-action="click->media#toggleLeft">
+      <button data-action="click->media#toggleLeft" aria-label="Display left menu" aria-controls="left-drawer">
```

Comment:
Is calling out "left" important here for someone who may be visually impaired? Is there a better description of this drawer we could use @alundgard ?